### PR TITLE
5.6 pairing

### PR DESCRIPTION
**Pull request by Clove210**

### CODE REVIEW

**Comment by Clove210** [comment]
@Bontgoy

File: ruby/new_alias_manager.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		elsif letter == "z"
+			letter = "a"
+		elsif consonants.include?(letter)
+			consonants.index(letter)
+			consonants[consonants.index(letter) +1]
+		else
+			letter
+        end	
+        end
+	end
```
Comment (Bontgoy) [inline ruby/new_alias_manager.rb]: It looks like you have an extra end here.  Is this file running properly?

File: ruby/new_alias_manager.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+#Release 1 And 2
+#Create a user interface
+#Ask the user to enter a name
+#return a fake name to them 
+#loop through the name scrambler until user enters in quit
+
+def name_scrambler
+	puts "Please enter a spy name?"
+	input = gets.chomp
```
Comment (Bontgoy) [inline ruby/new_alias_manager.rb]: lets keep `gets.chomp` out of our methods.  user input should be aloud in driver code but methods should have parameters where the input can be passed in as arguments.

File: ruby/new_alias_manager.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	scramble_name = input.split("").shuffle.join
```
Comment (Bontgoy) [inline ruby/new_alias_manager.rb]: we didn't ask to shuffle the characters.  I would like to see the characters changed and/or reversed.

Comment (Clove210) [inline ruby/new_alias_manager.rb]: I guess i didn't understand i thought it meant to just change up the letters. with the comments do I adjust the codes and then re- submit it?

**Pull request merged**